com.unity.shadergraph/Editor/GraphUI/GraphElements/ModelUI/GraphDataNode.cs

```diff
@@ -200,5 +156,127 @@ protected override void BuildContextualMenu(ContextualMenuPopulateEvent evt)
 
             base.BuildContextualMenu(evt);
         }
+
+        private IModelViewPart GetPartForPortField(
```

Comment:
Looks like this is just for GraphType, might have that be more explicit in the name unless it's going to change.